test/modules/standard/memory/countMemory/countMemory.prediff

```diff
@@ -14,12 +15,20 @@ if (-r $memfile) {
     }
   }
 } else {
-    $platform = `$ENV{CHPL_HOME}/util/chplenv/chpl_platform.py --target`; chomp($platform);
     if ($platform eq "darwin") {
         $memFreeInBytes = `sysctl -n hw.memsize`; chomp($memFreeInBytes);
     }
 }
 
+# cygwin32 running on a 64 bit windows can have more than 4GB reported by
+# /proc/meminfo, but only ~4GB is actually accessible
+if ($platform eq "cygwin32") {
+    $maxMemInBytes = 0xffffffff;
+    if ($memFreeInBytes > $maxMemInBytes) {
+        $memFreeInBytes = $maxMemInBytes;
+    }
```

Comment:
This makes me a little uneasy, though not terribly - I think it's the right choice, because I should hope that cygwin32 will become less and less relevant as time goes on and machines increase in size, and because 32 bits does imply the upper limit we're using here.  Still feels like hard-coding it, though.